libs/functions/calculate_projections.py

```diff
@@ -108,46 +119,57 @@ def get_state_projections_df(input_dir, initial_intervention_type, state_interve
     """
     for each state in our data look at the results we generated via run.py
     to create the projections
+
+    columns=CALCULATED_PROJECTION_HEADERS_STATES
     """
 
-    # save results in a list of lists, converted to df later
-    results = []
-    for state in list(US_STATE_ABBREV.values()):
-        intervention_type = _get_intervention_type(initial_intervention_type, state, state_interventions_df)
+    sdf = pd.DataFrame(US_STATE_ABBREV.values(), columns=['state'])
+    sdf.loc[:,'intervention_type'] = sdf.state.parallel_apply(
+        lambda x: _get_intervention_type(initial_intervention_type, x, state_interventions_df)
+        )
+    sdf.loc[:, 'path'] = sdf.parallel_apply(
+        lambda x: get_file_path(input_dir, x.state, x.intervention_type, fips=None),
+        axis=1).values
+    ndf = sdf.parallel_apply(
```

Comment:
What does ndf stand for?

Reply:
it was the previous df name in counties.  I can call it `new_df`